Even with introspection disabled clutter-box2d would try to install the gir
files.
This patch ignored the while introspection related section of the Makefile if
introspection is disabled.

JL 28/03/11
josh@linux.intel.com

Upstream-Status: Inappropriate [configuration] 

Index: clutter-box2d-0.10.0/clutter-box2d/Makefile.am
===================================================================
--- clutter-box2d-0.10.0.orig/clutter-box2d/Makefile.am
+++ clutter-box2d-0.10.0/clutter-box2d/Makefile.am
@@ -41,6 +41,8 @@ EXTRA_DIST = \
 
 BUILT_GIRSOURCES =
 
+if HAVE_INTROSPECTION
+
 ClutterBox2D-@CLUTTER_BOX2D_API_VERSION@.gir: $(INTROSPECTION_SCANNER) libclutter-box2d-@CLUTTER_BOX2D_API_VERSION@.la
 	$(QUIET_GEN)$(INTROSPECTION_SCANNER) -v \
 		--namespace ClutterBox2D --nsversion=@CLUTTER_BOX2D_API_VERSION@ \
@@ -70,3 +72,5 @@ typelibs_DATA = $(BUILT_GIRSOURCES:.gir=
 
 CLEANFILES=
 CLEANFILES += $(BUILT_GIRSOURCES) $(typelibs_DATA)
+
+endif
\ No newline at end of file
